timemachine/fe/atom_mapping.py

```diff
@@ -451,8 +430,9 @@ def leaf_filter_fxn(trial_core) -> bool:
 
         # distance score
         r2_ij = np.sum(np.power(r_i - r_j, 2))
-        rmsd = np.sqrt(r2_ij / len(core))
-        dists.append(rmsd)
+        # No need to perform square root here, only care about ordering
+        mean_sq_dist = r2_ij / len(core)
+        dists.append(mean_sq_dist)
```

Comment:
nit: rename `dists` to `mean_sq_dists`?

Reply:
Renamed in https://github.com/proteneer/timemachine/pull/1362/commits/4eeb264d7504193e4981f41ce4ac25634c642781